store/layered_store_hints.go

```diff
@@ -9,3 +9,23 @@ const (
 	LSH_NO_CACHE LayeredStoreHint = iota
 	LSH_MASTER_ONLY
 )
+
+func hintsContains(hints []LayeredStoreHint, contains LayeredStoreHint) bool {
+	for _, hint := range hints {
+		if hint == contains {
+			return true
+		}
+	}
+	return false
+}
+
+func hintsContainsAny(hints []LayeredStoreHint, contains ...LayeredStoreHint) bool {
```

Comment:
Actually. I like this for a different reason. @coreyhulen Doesn't like the variadic so I can change it to a bitset. Will take a look at doing that.